### ISSUE TITLE
Add branch name to neeto_emails, neeto_commons, and org_incineration

### ISSUE THREAD

**Issue by ghousemohamed** [bigbinary/neeto-compliance#103]
In the Gemfile for the gems we have included we need to include the branch name like so:

gem "neeto_emails", git: "https://github.com/bigbinary/neeto_emails", branch: "master"

cc: @unnitallman

**Comment by unnitallman** [comment]
@ghousemohamed Wouldn't `master` be taken as the default branch if none given?

**Comment by ghousemohamed** [comment]
@unnitallman Yes, but I've noticed that even after doing bundle update the commit id was nit changing for some repository. I had to do mention the branch and then had to do bundle update

**Comment by unnitallman** [comment]
@prasanthchaduvula _a Let us do this.
